Classes/UT3Hellbender.uc

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright © 2008, 2014 GreatEmerald
  * Copyright © 2008-2009 Wormbo
+ * Copyright © 2017 HellDragon (still too strong a word but if you want it, here you go)

```

Comment:
Again, no comments allowed in the copyright line.
While you're at it, you can also add `, 2017` to my line.

Reply:
Sure, do you want me to try and catch that on any others too along with GP?

Reply:
No. The year indicates when the person contributed. I contributed the way to add the function here, hence the year should be updated, but nowhere else.

Reply:
Also, don't update it like that, it needs to say `2008, 2014, 2017`!